bin/install-protobuf.sh

```diff
@@ -9,6 +9,9 @@ PLATFORM=`uname`
 echo $PLATFORM
 if [ "$PLATFORM" == 'Darwin' ]; then
    DOWNLOAD_URL="https://github.com/google/protobuf/releases/download/v$VERSION/protoc-$VERSION-osx-x86_64.zip"
+else
```

Comment:
I had to do this because I change the AMI for jenkins and that one does not have unzip

Reply:
Or maybe we don't want to change this script more than necessary :) It was just kind of weird to begin with.

Reply:
I like @timcharper recommendation